src/js/modules/articles/components/RecommendedRow.js

```diff
@@ -31,153 +56,46 @@ const styles = {
     marginBottom: "12px",
     textTransform: "uppercase",
   },
-  summary: {
-    color: "#000",
-    fontFamily: "Minion Pro",
-    fontSize: "14px",
-    fontWeight: "300",
-    lineHeight: "1.29",
-  },
-  titleWithImage: {
-    color: "#000",
-    fontFamily: "Minion Pro",
-    fontSize: "14px",
-    lineHeight: "1.21",
-    "&:hover, &:active, &:focus": {
-      color: "#000",
-    },
-  },
-  titleWithoutImage: {
-    color: "#000",
-    display: "block",
-    fontFamily: "Canela",
-    fontSize: "19px",
-    lineHeight: "1.38",
-    marginBottom: "2px",
-    "&:hover, &:active, &:focus": {
-      color: "#000",
-    },
-  },
-  figure: {
-    marginBottom: "4.1px",
-    maxHeight: "170px",
-    overflow: "hidden",
-    width: "100%",
-    "& img": {
-      width: "100%",
-    },
-  },
-  recommendedBlock: {
-    float: "left",
-    padding: "0 7px 0",
-    width: "25%",
-    "&:first-child": {
-      paddingLeft: 0,
-    },
-    "&:last-child": {
-      paddingRight: 0,
-    },
-  },
   "@media (max-width: 991px)": {
     RecommendedRow: {
       padding: "0 10%",
     },
   },
-  "@media (max-width: 767px)": {
+  "@media (max-width: 768px)": {
     RecommendedRow: {
       padding: "0 2%",
     },
-    figure: {
-      maxHeight: "270px",
-    },
-  },
-  "@media (max-width: 575px)": {
-    recommendedBlock: {
-      width: "50%",
-      marginBottom: "7px",
-    },
     recommendedList: {
-      /* recommendedBlocks are structured in a block form,
-           1  2
-           3  4,
-         rendering these next padding removals necessary.
-       */
-      "& > div:nth-child(2)": {
-        // 2nd recommendedBlock
-        paddingRight: "0 !important",
-      },
-      "& > div:nth-child(3)": {
-        // 3rd recommendedBlock
-        paddingLeft: "0 !important",
+      "& > .row > div": {
+        padding: "12px 0 !important",
       },
     },
   },
 };
 
-const RecommendedRow = ({ classes, media, articles, sections }) => {
-  // section is always the parent section, articles are always in that tree.
-  const recommendedArticles = articles.slice(0, 4);
+const RecommendedRow = ({ classes, data }) => {
```

Comment:
This isn't a code comment as much as a question. Where do you use RecommendedRow?